roles/bats/tasks/main.yml

```
@@ -31,7 +31,7 @@
 
 - block:
     - name: "Run bats"
-      shell: "bats --tap {{ item }} > {{ bats_output_dir }}/{{ item }}.tap"
+      shell: "bats -x --tap {{ item }} > {{ bats_output_dir }}/{{ item }}.tap"
```

```suggestion
      shell: "bats --trace --tap {{ item }} > {{ bats_output_dir }}/{{ item }}.tap"
```

long opts are easier to read ;)